docs/apache-airflow/security/api.rst

```diff
@@ -84,7 +84,7 @@ principal exists in the keytab file.
 Basic authentication
 ''''''''''''''''''''
 
-`Basic username password authentication <https://tools.ietf.org/html/rfc7617
+`Basic username password authentication <https://tools.ietf.org/html/rfc7617?
 https://en.wikipedia.org/wiki/Basic_access_authentication>`_ is currently
```

Comment:
The url isn't broken, and from the diff have only added a `?`

Reply:
I think those are two URLS that should be added next to each other - right now the second one was swallowed as the parameter of the first one.